utils/Bootstrap.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jest/no-conditional-expect */
+/*
+ *
+ * @group bootstrap
```

Comment:
why this? this is not a test. plz be careful

Reply:
Changed